README.md

```diff
@@ -166,9 +166,9 @@ A shared resource is helpful for both sides.
 
 ## Roadmap
 
-### The Holloway reader
+### The Holloway Reader
 
-The Holloway reader you’re using now is designed to help you find and navigate the
+The Holloway Reader you’re using now is designed to help you find and navigate the
 material you need. **Use the search box**. It will reveal definitions, section-by-section
```

Comment:
@jlevy we can add directions on HOW to use the search box here, too. What do you think, at least as some extra help for readers before search function becomes more obvious?

Reply:
Sure, yes, let's consider as a separate PR.